media-tv/plex-media-server/plex-media-server-1.16.4.ebuild

```diff
@@ -0,0 +1,157 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 )
+inherit eutils systemd unpacker pax-utils python-single-r1
+
+MINOR_VERSION="1469-6d5612c2f"
+
+_APPNAME="plexmediaserver"
+_USERNAME="plex"
+_SHORTNAME="${_USERNAME}"
+_FULL_VERSION="${PV}.${MINOR_VERSION}"
+
+URI="https://downloads.plex.tv/plex-media-server-new"
+
+DESCRIPTION="A free media library that is intended for use with a plex client."
+HOMEPAGE="https://www.plex.tv/"
+SRC_URI="amd64? ( ${URI}/${_FULL_VERSION}/debian/plexmediaserver_${_FULL_VERSION}_amd64.deb )"
+SLOT="0"
+LICENSE="Plex"
+RESTRICT="bindist strip"
+KEYWORDS="-* ~amd64"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+DEPEND="
+	sys-apps/fix-gnustack
+	dev-python/virtualenv[${PYTHON_USEDEP}]"
+
+RDEPEND="
+	net-dns/avahi
+	acct-user/plex
+	acct-group/plex
+	${PYTHON_DEPS}"
+
+QA_DESKTOP_FILE="usr/share/applications/plexmediamanager.desktop"
+QA_PREBUILT="*"
+QA_MULTILIB_PATHS=(
+	"usr/lib/${_APPNAME}/.*"
+	"usr/lib/${_APPNAME}/Resources/Python/lib/python2.7/.*"
+)
+
+EXECSTACKED_BINS=( "${ED}/usr/lib/plexmediaserver/libgnsdk_dsp.so*" )
+BINS_TO_PAX_MARK=( "${ED}/usr/lib/plexmediaserver/Plex Script Host" )
+
+S="${WORKDIR}"
+PATCHES=( "${FILESDIR}/virtualenv_start_pms_2019.patch" )
+
+src_unpack() {
+	unpack_deb ${A}
+}
+
+src_install() {
+	# Move the config to the correct place
+	local CONFIG_VANILLA="/etc/default/plexmediaserver"
```

Comment:
local variables are written with smallcase lettering, to differentiate from GLOBAL ones which shouldn't be modified within these phases.